src/main.rs

```diff
@@ -177,7 +177,7 @@ fn flash(matches: &ArgMatches) -> Result<(), Error>
     // Default template: `{wide_bar} {pos}/{len}`.
     let progress_bar = ProgressBar::new(file_size as u64)
         .with_style(ProgressStyle::default_bar()
-            .template(" {percent:>3}% |{bar:50}| {bytes}/{total_bytes} [{binary_bytes_per_sec} {elapsed}]")
+            .template(" {percent:>3}% |{bar:50}| {bytes}/{total_bytes} [{binary_bytes_per_sec} {elapsed}]").unwrap()
```

Comment:
This is not a change request for this PR, but a general note moving forward. We should work towards getting rid of unwraps and handling them gracefully instead. Let's try to avoid bmputil from panicking if we can avoid it. :D